modules/ROOT/pages/6.3-release-notes.adoc

```diff
@@ -0,0 +1,704 @@
+
+==== Added correct icons in dialog alert panel for each severity level
+
+In {productname} 6.3, improvements for the accessibility checker were added to provide additional support for users when trying to identify different alerts such as _info_, _warning_ or _error_ icons.
+
+In previous versions of {productname} the accessibility checker used the same triangle warning icon for all alerts when displayed. In {productname} 6.3, each alert type now displays a unique icon.
+
+* Warning alerts now display as an exclamation mark nested inside a yellow circle.
+* Error alerts now display as an exclamation mark nested inside a red triangle.
+* Success alerts now display as a green tick.
+
+For information on the **Accessibility Checker** plugin, see: xref:a11ychecker.adoc[Accessibility Checker].
+
+=== Advanced Code Editor 3.1.0
+
+The {productname} 6.3 release includes an accompanying release of the **Advanced Code Editor** premium plugin.
+
+**Advanced Code Editor** 3.1.0 includes the following improvement.
+
+
+==== new option to display the Advanced Code Editor inline instead of in a dialog
+
+In {productname} 6.3, the Advanced Code Editor plugin includes a new option, `advcode_inline`, that allows users to open the Advanced Code Editor within TinyMCE’s existing editor space instead of been displayed in a seperate dialog box.
+
+Using advcode inside TinyMCE means users on mobile devices or with otherwise narrow screens do not require a ToggleView button, as advcode forces the editor to open within the existing editor space in inline mode.
+
+The basic setup for this new option as follows.
+
+[source,js]
+----
+tinymce.init({
+  selector: "textarea", // change this value according to your HTML
+  plugins: "advcode",
+  advcode_inline: true, // To use this option, set this value to "true".
+  toolbar: "code",
+});
+----
+
+=== Checklist 2.0.2
+
+The {productname} 6.3 release includes an accompanying release of the **Checklist** premium plugin.
+
+**Checklist** 2.0.2 includes one fix:
+
+==== Checklist toggling did not preserve selection when used on an existing list.
+
+In previous {productname} versions, when selecting content over either an ordered or unordered list, then switching the list to a checklist, the user’s content selection was not preserved.
+
+In {productname} 6.3, the user selection is now preserved when toggling a list to a checklist.
+
+For information on the **Checklist** plugin, see xref:checklist.adoc[Checklist].
+
+=== Comments 3.2.0
+
+The {productname} 6.3 release includes an accompanying release of the **Comments** premium plugin.
+
+**Comments** 3.2.0 includes one fix and one change:
+
+==== A comment could be inserted when in an empty editor or block
+
+In previous {productname} versions a bug existed that allowed comments to be placed into an empty editor or block.
+
+With this update, the comments button, menu-item, comment sidebar and shortcut keys are now disabled when the editor contains no content.
```

Comment:
Speaking of terms-of-art, I wonder if **selection** is being used as such here.

Because this very document contains a sub-section — _The selection now expands to encompass the word containing the insertion point when a comment is added_ — which I read as meaning the comment button, meni-item, sidebar, and shortcut keys can be active when there is nothing selected in the editor.

And the change above is about no longer being able to add a comment when the editor contains nothing a user can select.